### DISCUSSION TITLE
Where  can I get the public key...

### DISCUSSION THREAD

**Discussion by chris2553** [OpenPrinting/cups#327] [General] [answered]
... to verify the authenticity of cups-2.4.1-source.tar.gz, please?

$ gpg --auto-key-retrieve  --verify cups-2.4.1-source.tar.gz.sig 
gpg: assuming signed data in 'cups-2.4.1-source.tar.gz'
gpg: Signature made Thu 27 Jan 2022 11:41:15 GMT
gpg:                using EDDSA key 7082A0A50A2E92640F3880E0E4522DCC9B246FF7
gpg: Can't check signature: No public key

**Comment by zdohnal** [comment, marked as answer]
Hi @chris2553 ,

I uploaded my new key to pgp.mit.edu via gpg command several hours ago, but it seems it didn't work correctly - now I've put the key itself via web ui to pgp.mit.edu, keyserver.ubuntu.com and keys.openpgp.org, so hopefully it will be fixed now.

**Reply by chris2553** [reply]
Yep, Got it. Thanks
